src/align_stack.cpp

```diff
@@ -296,14 +297,11 @@ void AlignStack::Add(Chunk *start, size_t seqnum)
          tmp = tmp->GetNext();
       }
 
-      if (  (  chunk_is_star(tmp)
-            && m_star_style == SS_DANGLE)
-         || (  chunk_is_addr(tmp)
-            && m_amp_style == SS_DANGLE)
-         || (  chunk_is_nullable(tmp)
-            && (m_star_style == SS_DANGLE))
-         || (  chunk_is_msref(tmp)
-            && m_star_style == SS_DANGLE)) // TODO: add m_msref_style
+      if (  tmp != nullptr
+         && (  (tmp->IsStar() && m_star_style == SS_DANGLE)
+            || (chunk_is_addr(tmp) && m_amp_style == SS_DANGLE)
+            || (chunk_is_nullable(tmp) && (m_star_style == SS_DANGLE))
+            || (chunk_is_msref(tmp) && m_star_style == SS_DANGLE))) // TODO: add m_msref_style
```

Comment:
The project maintainer (@guy-maurel) prefers to have the various conditions on separate lines.